groups/rds/data.tf

```diff
@@ -29,5 +29,5 @@ data "aws_subnet_ids" "placement" {
 
 data "aws_route53_zone" "selected" {
   name         = local.dns_zone_name
-  private_zone = false
+  private_zone = local.dns_zone_isprivate
```

Comment:
At the risk of being pedantic can we keep this consistent and go with `dns_zone_is_private` for legibility sake